docs/source/user-guide/nlp/nlp_properties.rst

```diff
@@ -0,0 +1,167 @@
+.. _nlp_properties_guide:
+
+=================
+NLP Properties
+=================
+
+Properties are one-dimension values that are extracted from either the text, labels or predictions. For example, a
```

Comment:
Right now we don't token properties, and addressing it complicates this guide significantly. I would advise for now just talking about text properties.

Reply:
Also you can't really configure them even when we will implement, so maybe no need to add them here.

Reply:
Not sure how will be done, but we are planning to do this before the release...

Reply:
That's a good point. Will change